src/main/scala/streaming/StreamingWithLogging.scala

```diff
@@ -0,0 +1,32 @@
+package streaming
+
+import com.danielasfregola.twitter4s.TwitterStreamingClient
+import com.danielasfregola.twitter4s.entities.Tweet
+import com.danielasfregola.twitter4s.processors.TwitterProcessor._
+import com.typesafe.scalalogging.LazyLogging
+
+
+object StreamingWithLogging extends App with LazyLogging {
+
+  // Make sure to define the following env variables:
+  // TWITTER_CONSUMER_TOKEN_KEY and TWITTER_CONSUMER_TOKEN_SECRET
+  // TWITTER_ACCESS_TOKEN_KEY and TWITTER_ACCESS_TOKEN_SECRET
+  val streamingClient = TwitterStreamingClient()
+
+  // change this to whichever words you want to track
+  val trackedWords = Seq("#scala", "#functionalprogramming", "#haskell")
+
+  // logger is accessible with trait LazyLogging - see https://github.com/typesafehub/scala-logging
+  // backend used in project is logback - https://logback.qos.ch/
+  // configuration for logback is logback.xml in resources
+  logger.info(s"Launching streaming session with tracked keywords: $trackedWords")
+
+  // demo purpose only, do not use logging to serialize tweets :)
+  streamingClient.filterStatuses(tracks = trackedWords) {
+    case tweet: Tweet =>
+      // will log to console Tweet text - check TwitterProcessor for details
+      // will log to file StreamingClient debug message because debug level is enabled in logback.xml
```

Comment:
Having the debug enabled for all the examples is a bit too much: people will be bombarded by messages on their console. Can we enable the debugger only for this file and leave it as info for the rest? I believe we can :)

Reply:
Even with the DEBUG level on, it should be only output INFO messages (tweet text) on the console thanks to the LevelFilter setting in logback.xml, set to match only INFO.

I tested it but it was late, I will double check :)

Reply:
I see the problem now! you need to tell logback to enable the debugger in twitter4s, since `log.debug` is defined in twitter4s and not in the twitter4s-demo example.

Note that because your twitter4s PR hasn't been merged yet, you need to either publish it locally or wait for it to be merged (I am planning to do it this afternoon)

Reply:
Actually, I was wrong! you are correct: the filtering is making sure the logs are working as expected